content/docs/command-reference/exp/save.md

```diff
@@ -0,0 +1,63 @@
+# exp save
+
+Save the current project status as a [DVC experiment].
+
+[dvc experiment]: /doc/user-guide/experiment-management/experiments-overview
+
+## Synopsis
+
+```usage
+usage: dvc exp save [-h] [-q | -v] [-f]
+                   [--json] [-n <name>]
+                   [-I path]
+```
+
+## Description
+
+Provides a way to save the current status of your <abbr>project</abbr> as an
+<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+Only files tracked by either Git or DVC are saved to the experiment. Use the
+`--include-untracked` (`-I`) option to explicitly include any untracked files in
+the experiment.
+
+[Review] your experiments with `dvc exp show`. Successful ones can be [made
+persistent] by restoring them via `dvc exp branch` or `dvc exp apply` and
+committing them to the Git repo. Unnecessary ones can be [cleared] with
+`dvc exp gc`.
+
+[review]: /doc/user-guide/experiment-management/comparing-experiments
+[made persistent]: /doc/user-guide/experiment-management/persisting-experiments
+[cleared]: /doc/user-guide/experiment-management/cleaning-experiments
+
+## Options
+
+- `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
+  default one will be generated otherwise, such as `exp-f80g4` (based on the
+  experiment's hash).
+
+- `-f`, `--force` - overwrite the experiment if an experiment with the same name
+  already exists.
+
+- `-h`, `--help` - prints the usage/help message, and exits.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if all
+  stages are up to date or if all stages are successfully executed, otherwise
+  exit with 1. The command defined in the stage is free to write output
+  regardless of this flag.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+[unique name]:
+  https://dvc.org/doc/user-guide/experiment-management/experiments-overview#how-does-dvc-track-experiments
+
+## Examples
```

Comment:
Maybe we could add an example illustrating the usage of `-I`

Reply:
Wrt this: I think `-I` is already illustrated in the existing example, and I updated the option desc a little in #4200.